README.md

```diff
@@ -46,35 +56,247 @@ There are 3 fun ways for installing this plugin.
+| Checkbox                  | m-input:checkbox               |
+| Date Picker               | m-input:date                   |
+| File Input                | m-input:file                   |
+| Radio Button              | m-input:radio                  |
+| Radio Button with Gap     | m-input:radio-gap              |
+| Range Input               | m-input:range                  |
+| Select Box-disabled       | m-input:select-disabled        |
+| Switch-enabled            | m-input:switch-enabled         |
+| Switch-disabled           | m-input:switch-disabled        |
+
+### Badges
+
+| Component                 | Snippet code                   |
+|---------------------------| :-----------------------------:|
+| Badge(old)                | m-badge                        |
+| New badge                 | m-badge:new                    |
+
+### Buttons
+
+| Component                 | Snippet code                   |
+|---------------------------| :-----------------------------:|
+| Button-disabled           | m-button:disabled              |
+| Fixed Floating Button     | m-fixed-floating-button        |
+| Flat Button               | m-button:flat                  |
+| Floating Button           | m-button:floating              |
+| Large Button              | m-lg-button                    |
+| Raised Button             | m-raised-button                |
+| Raised Button Icon        | m-raised-button:icon           |
+| Submit Button             | m-submit-button                |
 
 ### Cards
 
 | Component                 | Snippet code                   | 
 |---------------------------| :-----------------------------:| 
 | Default card              | m-card                         | 
-| Image card                | m-card:image                   |
\ No newline at end of file
+| Image card                | m-image-card                   |
+| Panel card                | m-panel-card                   |
+| Revile card               | m-revile-card                  |
+| Default card(large)       | m-card:lg                      |
+| Default card(medium)      | m-card:md                      |
+| Default card(small)       | m-card:sm                      |
+| Revile card(large)        | m-revile-card:lg               |
+| Revile card(medium)       | m-revile-card:md               |
+| Revile card(small)        | m-revile-card:sm               |
+
+### Collapsible
+
+| Component                   | Snippet code                 |
+|-----------------------------| :---------------------------:|
+| Collapsible(single select)  | m-collapsible                |
+| Collapsible(multiple select)| m-collapsible:expandable     |
+| Collapsible(default active) | m-collapsible:selected       |
+
+### Collections
+
+| Component                   | Snippet code                 |
+|-----------------------------| :---------------------------:|
+| Basic Collection            | m-collection                 |
+| Collection with avatar      | m-avatar-collection          |
```

Comment:
Cool



On Thu, Apr 9, 2015 at 6:46 PM, Barnabas A Nsoh <notifications@github.com>
wrote:

> In README.md
> <https://github.com/ayinloya/materialized-css-snippets/pull/2#discussion_r28090546>
> :
>
> > +| Revile card(small)        | m-revile-card:sm               |
> > +
> > +### Collapsible
> > +
> > +| Component                   | Snippet code                 |
> > +|-----------------------------| :---------------------------:|
> > +| Collapsible(single select)  | m-collapsible                |
> > +| Collapsible(multiple select)| m-collapsible:expandable     |
> > +| Collapsible(default active) | m-collapsible:selected       |
> > +
> > +### Collections
> > +
> > +| Component                   | Snippet code                 |
> > +|-----------------------------| :---------------------------:|
> > +| Basic Collection            | m-collection                 |
> > +| Collection with avatar      | m-avatar-collection          |
>
> to make the snippet consistent this could be *m-collection:avatar*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ayinloya/materialized-css-snippets/pull/2/files#r28090546>
> .
>